test/unit_tests/braket/circuits/test_angled_gate.py

```diff
@@ -44,3 +48,11 @@ def test_getters():
 @pytest.mark.xfail(raises=AttributeError)
 def test_angle_setter(angled_gate):
     angled_gate.angle = 0.14
+
+
+def test_np_float_angle_json():
+    angled_gate = AngledGate(angle=np.float32(0.15), qubit_count=1, ascii_symbols=["foo"])
+    angled_gate_json = BaseModel.construct(target=[0], angle=angled_gate.angle).json()
+    match = re.match(r'\{"target": \[0], "angle": (\d*\.?\d*)}', angled_gate_json)
+    angle_value = float(match.group(1))
+    assert np.isclose(angle_value, 0.15)
```

Comment:
```suggestion
    assert np.isclose(angle_value, angled_gate.angle)
```

Reply:
I like the suggestion. Following this path, the rounding error is already accounted for in the float32 -> float conversion, so we would expect angle_value to match angle_gate.angle exactly, I'll push one more commit removing isclose()